app/models/tweet.rb

```diff
@@ -48,6 +48,8 @@ def fetch(status_id)
       retrieve_and_save_tweet(status_id)
     rescue TwitterClient::Errors::NotFound => e
       raise e, I18n.t("models.tweet.tweet_not_found")
+    rescue StandardError => e
```

Comment:
it's possible we want a tighter match here...
https://www.rubydoc.info/gems/twitter/Twitter/REST/Tweets#status-instance_method suggests the errors _it_ will raise are Twitter::Error::Unauthorized, and Twitter::Error::Forbidden, but we wrap those in https://github.com/forem/forem/blob/main/app/services/twitter_client/client.rb#L47-L53 (I assume those are both ClientError conditions

Maybe only rescue and relabel TwitterClient::Errors::ClientError (if something else goes wrong, like a network connection issue, warning about authentication seems misleading)

Alternately, does it make sense to move this decision into the TwitterClient::Client (adding an authentication error, and handling Unauthorized specifically?)

Reply:
ok

Reply:
follow up documenting the work - testing live led to the discovery that invalid or absent credentials result in a bad request error (not an unauthorized?) 

This might be based on a format check, like "is the secret the right size, is the key the right size", and I don't know if we ever get a 401 response (it feels like we should) but it would be easy to add that if it were observed.